integration-tests/opentelemetry.spec.js

```diff
@@ -121,15 +121,15 @@ describe('opentelemetry', () => {
         OTEL_LOG_LEVEL: 'debug',
         DD_TRACE_SAMPLE_RATE: '0.5',
         OTEL_TRACES_SAMPLER: 'traceidratio',
-        OTEL_TRACES_SAMPLER_ARG: '0.1',
+        OTEL_TRACES_SAMPLER_ARG: '1.0',
```

Comment:
If I'm understanding this correctly, we're incrementing the `otelHiding.length` in the test because now we consider the `OTEL_TRACES_SAMPLER_ARG` config as another config that's being overlooked by its corresponding Datadog config, `DD_TRACE_SAMPLE_RATE`?

Reply:
Yes, since they both map to DD_TRACE_SAMPLE_RATE it makes more sense to increment them as individual pairs of tags. since a user could have inputted an invalid OTEL_TRACES_SAMPLER value & and an OTEL_TRACES_SAMPLER_ARG value or either or